base/abstractarray.jl

```diff
@@ -2196,8 +2196,11 @@ function _typed_hvncat(::Type{T}, ::Val{N}, as...) where {T, N}
     # into the destination
     nd = N
     Ndim = 0
-    for a ∈ as
+    for i ∈ eachindex(as)
+        a = as[i]
```

Comment:
Why this change? Doesn't seem like `i` is used anywhere else.

Reply:
For the error message to specify the index. Guess I could have done `enumerate()`

Reply:
Ah, I didn't see that. In that case it's fine.